icons/gender-trans.svg

```diff
@@ -0,0 +1,20 @@
+<svg xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round">
+  <path d="M16 1H20C20.5523 1 21 1.44772 21 2V6" />
+  <path d="M1 6L1 2C1 1.44772 1.44772 1 2 1L6 1" />
+  <path d="M15 7L20.5 1.5" />
+  <path d="M1.5 1.5L7 7" />
+  <circle cx="11" cy="11" r="5" />
+  <path d="M11 16V21" />
+  <path d="M8 19H14" />
+  <path d="M3 7L7 3" />
+</svg>
+
+
```

Comment:
This does not follow our design guidelines (1px padding, 2px gap between elements).

Reply:
Also, I think that since all our current arrows have sharp heads, so should these.